From 433402535ca1da1230c69587814e6bab38ae3fbf Mon Sep 17 00:00:00 2001
From: Ash Charles <ashcharles@gmail.com>
Date: Thu, 19 Dec 2013 12:40:52 +0100
Subject: [PATCH 8/8] [omap3][overo] Add Board Revision for Wireless chip

Gumstix Overo COMs with board revision 0x4 use a different Wifi and
Bluetooth module: Wi2Wi's W2CBW0015.  No other code changes are required
in u-boot---just handling of this particular board revision.

Signed-off-by: Ash Charles <ashcharles@gmail.com>
---
 board/overo/overo.c | 1 +
 board/overo/overo.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/board/overo/overo.c b/board/overo/overo.c
index 5e04adc..3e1bc16 100644
--- a/board/overo/overo.c
+++ b/board/overo/overo.c
@@ -147,6 +147,7 @@ void get_board_mem_timings(struct board_sdrc_timings *timings)
 		timings->rfr_ctrl = SDP_3430_SDRC_RFR_CTRL_165MHz;
 		break;
 	case REVISION_1: /* Micron 256MB/512MB, 1/2 banks of 256MB */
+	case REVISION_4:
 		timings->mcfg = MICRON_V_MCFG_200(256 << 20);
 		timings->ctrla = MICRON_V_ACTIMA_200;
 		timings->ctrlb = MICRON_V_ACTIMB_200;
diff --git a/board/overo/overo.h b/board/overo/overo.h
index 33ea58e..410fa3f 100644
--- a/board/overo/overo.h
+++ b/board/overo/overo.h
@@ -22,6 +22,7 @@ const omap3_sysinfo sysinfo = {
 #define REVISION_1	0x1
 #define REVISION_2	0x2
 #define REVISION_3	0x3
+#define REVISION_4	0x4
 
 /*
  * IEN  - Input Enable
-- 
1.8.3.2

